### DISCUSSION TITLE
register ip in log - improvement suggestion

### DISCUSSION THREAD

**Discussion by fabiofurlanc** [InternetConnectionMonitor/icm-public#114] [Ideas and Feature requests]
To record which link the connection was through when the internet went down, it would be interesting to record the IP that was connected before it went down.

**Comment by Shane8johnson** [comment]
I Second the need for this. Having a primary and a backup ISP, with auto failover, it would be nice to be able to see which one is which in the log, and I think the easiest way to do that is via a setting to log the IP along with the online/offline time.

**Comment by dmishh** [comment]
Will be implemented in #154
